Templates/ManualTrigger/index.js

```diff
@@ -1,4 +1,4 @@
-﻿module.exports = function (input, context) {
-    context.log('Node.js manually triggered function called with input ' + input);
+﻿module.exports = function (context, input) {
+    context.log('Node.js manually triggered function called with input ' + JSON.stringify(input));
```

Comment:
You don't need to if it is a string. However, if your input object is a JSON, it will log "[Object object]". This has always been the case; our samples just always assumed string for input data type. I figured we should handle both strings and JSON.